pkg/p2p/libp2p/internal/handshake/handshake_test.go

```diff
@@ -178,7 +177,7 @@ func TestHandshake(t *testing.T) {
 
 func TestHandle(t *testing.T) {
 	nodeInfo := Info{
-		Address:   "node1",
+		Address:   []byte("node1"),
```

Comment:
Use swarm.Address not []byte as type, and construct it from actual byteslice or use swarm.NewAddress if it is constructed from hex-encoded string.